code/src/components/app-table/components/app-table-body/AppTableBody.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+
+import AppTableBody from "@/components/app-table/components/app-table-body/AppTableBody";
+
+describe("AppTableBody", () => {
+  test("renders with children", () => {
+    const { getByText } = render(
```

Comment:
Why do you decide to use `getByText` with `render` instead `screen` ?

Reply:
changed to `sceen`